### PR TITLE
docs(README): add csrf and loadbalance description

### PR DESCRIPTION
**Pull request by L2ncE**
#### What type of PR is this?

docs

#### Check the PR title.

- [x] This PR title match the format: \<type\>(optional scope): \<description\>
- [x] The description of this PR title is user-oriented and clear enough for others to understand.

#### (Optional) Translate the PR title into Chinese.

在 README 中新增 csrf 与 loadbalance 拓展的描述。

#### Which issue(s) this PR fixes:

None.

### CODE REVIEW

File: README_cn.md [hunk 1]
```diff
@@ -58,32 +58,34 @@ Hertz[həːts] 是一个 Golang 微服务 HTTP 框架，在设计之初参考了
...
+| [Basicauth](https://github.com/cloudwego/hertz/tree/develop/pkg/app/middlewares/server/basic_auth) | Basicauth 中间件能够提供 HTTP 基本身份验证。                                                                    |
+| [Lark](https://github.com/hertz-contrib/lark-hertz)                                                | 在 Hertz 中处理 Lark/飞书的卡片消息和事件的回调。                                                                   |
+| [Logger](https://github.com/hertz-contrib/logger)                                                  | Hertz 的日志拓展，提供了对 zap、logrus、zerologs 日志框架的支持。                                                     |
+| [Csrf](https://github.com/hertz-contrib/csrf)                                                      | Csrf 中间件用于防止跨站点请求伪造攻击。                                                                            |
+| [Loadbalance](https://github.com/hertz-contrib/loadbalance)                                        | 提供适用于 Hertz 的负载均衡算法。                                                                              |
```
Comment (li-jin-gou) [inline README_cn.md:88]: Looks good, but why has the formatting all changed, shouldn't it normally just add two lines of introduction?

**Review by li-jin-gou — CHANGES_REQUESTED** [review]

File: README_cn.md (hunk 1 above)
Comment (L2ncE) [reply README_cn.md:88]: > Looks good, but why has the formatting all changed, shouldn't it normally just add two lines of introduction?

for formatting

Comment (L2ncE) [reply README_cn.md:88]: Normally this is not the case.Because the description of the registry extension is too long, it needs to be adapted

Comment (li-jin-gou) [reply README_cn.md:88]: OK

**Review by li-jin-gou — APPROVED** [review]

**Review by welkeyever — APPROVED** [review]

**Pull request merged**